locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -141,7 +144,10 @@ FROM node:boron
 WORKDIR /usr/src/app
 
 # Install app dependencies
-COPY package.json package-lock.json .
+COPY package.json .
+# For npm@5 or later, copy package-lock.json as well
```

Comment:
Ok, then how about doing the same for line 103/105? Or am I missing something?
It doesn't really matter, it just triggers my OCD 😄.